tests/acceptance/features/coreApiSharePublicLink1/createPublicLinkShare.feature

```diff
@@ -59,10 +59,11 @@ Feature: create a public link share
       | uid_owner              | %username%      |
       | name                   |                 |
     And the public should be able to download the last publicly shared file using the new public WebDAV API with password "%public%" and the content should be "Random data"
-    And the HTTP status code should be "200"
-    And the public download of the last publicly shared file using the new public WebDAV API with password "%regular%" should fail with HTTP status code "401"
+    When the public downloads the last public link shared file with password "%regular%" using the new public WebDAV API
```

Comment:
do we have step like this?
```suggestion
    When the public tries to download the last public link shared file with password "%regular%" using the new public WebDAV API
```

Reply:
no we don't

Reply:
so then we can have one (multiple steps can be assigned to a single stef def)

Reply:
do you mean to make a new step

Reply:
Yep, we can link multiple steps like this:
```php
	/**
	 * @When /^the public downloads the last public link shared file with password "([^"]*)" using the (old|new) public WebDAV API$/
	 * @When /^the public tries to download the last public link shared file with password "([^"]*)" using the (old|new) public WebDAV API$/
	 *
```